### PR TITLE
Feature/v3.x.x/1002 lzma2 compression support

### PR DESCRIPTION
**Pull request by Dzejkop**

### CODE REVIEW

File: Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs [hunk 1]
```diff
@@ -215,45 +246,47 @@ private void UnpackRegularFile(Pack1Meta.FileEntry file, Action<double> onProgre
...
-                            targetStream.Write(buffer, 0, count);
-                            bytesProcessed += count;
-                            onProgress(bytesProcessed / (double) file.Size.Value);
-                        }
-                    }
-                    catch (Exception e)
-                    {
-                        DebugLogger.LogException(e);
-
-                        var logManager = PatcherLogManager.Instance;
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs]: Catching and reporting uncompression exception has been removed from here. Is it intentional?

Comment (Dzejkop) [reply Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs]: Intentional at the time but completely misguided, thanks for noticing.

Comment (witcher112) [reply Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs]: So it will be brought back right?

Comment (Dzejkop) [reply Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs]: Yup, I'll get to it today

**Pull request merged**